code/modules/admin/verbs/buildmode.dm

```diff
@@ -297,3 +298,33 @@
 				if(holder.throw_atom)
 					holder.throw_atom.throw_at(object, 10, 1)
 					log_admin("[key_name(usr)] threw [holder.throw_atom] at [object]",admin_key=key_name(usr))
+
+/verb/load_template()
+	set name = "Load Template"
+	set category = "Special Verbs"
+	if(!usr)
```

Comment:
You should also be checking admin rights here.

Reply:
I thought it should not be necessary, since this verb appears only when you press Toggle Build mode - which checks for perms already.

Reply:
It isn't necessary, but it's always better to make sure shit doesn't go wrong.